src/components/Application/ProgressBarNEW.tsx

```diff
@@ -114,7 +143,7 @@ const ProgressBarNEW: React.FC<ProgressBarProps> = ({ structure, requestRevalida
       },
       onTitleClick: () => handleChangeToPage(code, 1),
       content: {
-        content: getPageList(code, pages),
+        content: getPageList(code, pages, isIsStrictSection),
```

Comment:
```suggestion
        content: getPageList(code, pages, isIsStrictSection),
```
```suggestion
        content: getPageList(code, pages, isStrictSection),
```

Reply:
I did one global replace to make sure we capture all of the places this was mistyped: c7093b4